code/controllers/subsystem/processing/ai_movement.dm

```diff
@@ -5,7 +5,7 @@ PROCESSING_SUBSYSTEM_DEF(ai_movement)
 	priority = FIRE_PRIORITY_NPC_MOVEMENT
 	runlevels = RUNLEVEL_GAME | RUNLEVEL_POSTGAME
 	init_order = INIT_ORDER_AI_MOVEMENT
-	wait = 0.1 SECONDS
+	wait = 1
```

Comment:
why would you do this, this is blasphemy

Reply:
sugondese